src/ussd_public.js

```diff
@@ -368,12 +370,23 @@ go.app = function() {
                     new Choice('9_11', $("Between 9-11am")),
                     new Choice('2_5', $("Between 2-5pm"))
                 ],
-                next: 'state_voice_confirm'
+                next: function(choice) {
+                    return go.utils_project
+                        .update_msg_format_time(
+                            self.im,
+                            'voice',
```

Comment:
Better to change it before we start filling the system with data and have to migrate data. How long do you think it'll take?